packages/nocodb/src/lib/models/ProjectUser.ts

```diff
@@ -183,4 +183,8 @@ export default class ProjectUser {
       project_id: projectId,
     });
   }
+
+  static async getAllProjectsUsers(ncMeta = Noco.ncMeta) {
+    return await ncMeta.metaList2(null, null, MetaTable.PROJECT_USERS, {});
+  }
```

Comment:
1. I think it's better to call `list` (aligned with other models).
2. We can apply cache here. See the below `list` for reference.
https://github.com/nocodb/nocodb/blob/08d73677dc6e794babe930edd8b54e2dc465434f/packages/nocodb/src/lib/models/Plugin.ts#L44-L51
3. I saw there's a function called `getUsersList`. Can we somehow use it?

Reply:
doing the change for points 1 and 2.
For point 3 `getUsersList` finds users of a given project and not of all.
I did not want to query for each project so chose to get a list of projectUsers.